lib/Humming-Bird/Core.rakumod

```diff
@@ -300,15 +300,14 @@ my constant $PARAM_IDX = ':';
 class Route is Callable {
     has Str $.path;
     has &.callback;
+    has &.composition;
     has @.middlewares is Array; # List of functions that type Request --> Request
 
     method CALL-ME(Request:D $req) {
         my $res = Response.new(status => HTTP::Status(200));
         if @!middlewares.elems {
```

Comment:
Again... didn't actually change but be aware of how lazy arrays could interfere with the logic: one can't call `.elems` on them and therefore they would all turn into `False` with regards to the `if`. A plain Bool check might perform better (that tries to reify exactly one element to figure out if the list is empty or not) if you need to support laziness.